terraform/datadog/aws.tf

```diff
@@ -0,0 +1,35 @@
+resource "datadog_monitor" "ec2-cpu-credits" {
+  name               = "${format("%s EC2 CPU credits", var.env)}"
+  type               = "query alert"
+  message            = "Instance is low on CPU credits and may start to perform badly"
+  escalation_message = "Instance is out of CPU credits and may perform badly"
+  notify_no_data     = false
+  query              = "${format("avg(last_30m):avg:aws.ec2.cpucredit_balance{deploy_env:%s} by {bosh-job,bosh-index} <= 0", "master")}"
+
+  thresholds {
+    warning  = "20"
+    critical = "0"
+  }
+
+  require_full_window = true
+
+  tags = ["deployment:${var.env}", "service:${var.env}_monitors", "job:all"]
+}
+
+resource "datadog_monitor" "rds-cpu-credits" {
+  name               = "${format("%s RDS CPU credits", var.env)}"
+  type               = "query alert"
+  message            = "Instance is low on CPU credits and may start to perform badly"
+  escalation_message = "Instance is out of CPU credits and may perform badly"
+  notify_no_data     = false
+  query              = "${format("avg(last_30m):avg:aws.rds.cpucredit_balance{deploy_env:%s} by {bosh-job,bosh-index} <= 0", "master")}"
```

Comment:
I think it's important that monitors for the platform health don't get conflated with monitors relating to tenant services. Is there a way this could be restricted to the platform RDS instances (I think it's only the Bosh RDS instance at present that's a t2)?